Chapters/1-callouts.pillar

```diff
@@ -93,33 +94,33 @@ This call-out binding, a Pharo method, is called ==ticksSinceStart== and happens
 Indeed, uFFI does not impose any restrictions as far as how to call your external functions.  This can come in handy for decoupling your methods from underlying C-level implementation details.
 
 We invoke the C function using the Pharo method ==ffiCall:module:==, which is defined by uFFI.  We assemble the message arguments it needs by deconstructing the target C function declaration and referencing the name of the library in which it's defined.  uFFI takes the pieces and performs all the necessary work to make the call-out.  Continuing our example, we see that uFFI:
-- Locates the specified library (==libc==, ==msvcrt==, ...) in our host system,
```

Comment:
Pay attention because you remove may concrete information that is referring to that particular example and this is nice for the reader to see them

Reply:
I see what you mean.  However, there was one commit for Chapter 1 that was not in this branch originally.  I merged that commit into this branch (and closed the earlier pull request).  Now Chapter 1 reads quite a bit better, so you should probably re-read it.

Also, the paragraphs above this list already point out variations of the example for all 3 platform library types.  The main example that runs through all the sections in this chapter have been assuming a Linux host, and I wanted to keep that consistent.  The list originally might have been confusing by listing two libraries (libc & msvcrt).

I decided it would read better if I generalized the entire process, then returned to the specific example. The newly merged commit reflects this.  I did add another sentence to point out that the modules will almost always have different names for different host platforms.